tests/specs/api.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+var test = require('tape')
+
+var Store = require('../../')
+
+test('has "api" methods', function (t) {
+  t.plan(9)
+
+  var store = new Store('test-db')
+
+  t.is(typeof store.add, 'function', 'has "add" method')
+  t.is(typeof store.find, 'function', 'has "find" method')
+  t.is(typeof store.findOrAdd, 'function', 'has "findOrAdd" method')
+  t.is(typeof store.findAll, 'function', 'has "findAll" method')
+  t.is(typeof store.update, 'function', 'has "update" method')
+  t.is(typeof store.updateOrAdd, 'function', 'has "updateOrAdd" method')
+  t.is(typeof store.updateAll, 'function', 'has "updateAll" method')
+  t.is(typeof store.remove, 'function', 'has "remove" method')
+  t.is(typeof store.removeAll, 'function', 'has "removeAll" method')
```

Comment:
There is a check for `store.db` in the `constructor` spec. Do you want me to check again after the api is added?